Content.Server/UserInterface/ActivatableUISystem.cs

```diff
@@ -103,27 +122,6 @@ public void SetCurrentSingleUser(EntityUid uid, IPlayerSession? v, ActivatableUI
             RaiseLocalEvent(uid, new ActivatableUIPlayerChangedEvent(), false);
         }
 
-        public override void Update(float frameTime)
-        {
-            base.Update(frameTime);
-
-            foreach (var component in EntityManager.EntityQuery<ActivatableUIComponent>(true))
-            {
-                var ui = component.UserInterface;
-                if (ui == null) continue;
-                // Done to skip an allocation on anything that's not in use.
-                if (ui.SubscribedSessions.Count == 0) continue;
-                // Must ToList in order to close things safely.
-                foreach (var session in ui.SubscribedSessions.ToArray())
-                {
-                    if (session.AttachedEntity == null || !_actionBlockerSystem.CanInteract(session.AttachedEntity.Value))
-                    {
-                        ui.Close(session);
-                    }
-                }
-            }
-        }
-
```

Comment:
Ideally it gets closed instantly; with that being said we already need an actionblocker component for mover code that issues events whenever something changes so this is probably fine.

Reply:
Yeah it'd be nice if there was just some sort of "action-blocker state changed event"`" or something.